Makefile

```diff
@@ -123,15 +127,30 @@ ssj_sources=src/ssj/main.c \
    src/ssj/parser.c \
    src/ssj/session.c
 
+ifeq ($(os), Darwin)
+LINKER_ARGS=-Wl,-rpath,\$$ORIGIN
+engine_libs+=-lallegro_main
+CHAKRACORE_URL=https://aka.ms/chakracore/cc_osx_x64_1_11_15
```

Comment:
I didn't even realize this existed as an officially-supplied binary, Go figure.

Reply:
Yup :)